cdap-runtime-ext-remote-hadoop/src/main/java/io/cdap/cdap/runtime/spi/provisioner/remote/RemoteHadoopConf.java

```diff
@@ -33,14 +33,21 @@ public class RemoteHadoopConf {
   private final String initializationAction;
   private final String kerberosKeytabPath;
   private final String kerberosPrincipal;
+  private final EdgeNodeCheckType edgeNodeCheckType;
+  private final int timeout;
+
+  private static final int DEFAULT_CHECK_TIMEOUT = 5000;
 
   private RemoteHadoopConf(SSHKeyPair sshKeyPair, String host, @Nullable String initializationAction,
```

Comment:
@achernatsky-cybervision 
even if we can, how easy would be to "make" rest of the code to use it instead of the original .... as result would it worth it and what the sense of keeping the old one then?